Rb.Forms.Barcode.nuspec

```diff
@@ -11,25 +11,18 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes full of awesomeness and also unicorns. Please check the website for further details.
 
-Beta notice: Currently only android is supported.</description>
+Beta notice: Now with experimental iOS support.</description>
     <summary>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes.</summary>
-    <releaseNotes>0.5.2-beta:
+    <releaseNotes>0.6.0-beta: 
+* Add experimental iOS support.
```

Comment:
Please add a `new` in front. Is it experimental? Besides that this is why the package is still beta right? :)